### ISSUE TITLE
Defining Github issues and project management structure

### ISSUE THREAD

**Issue by izqui** [aragon/issues#73]

**Comment by luisivan** [comment]
I agree with everything. I'd leave aragon/aragon for the dapp, which right now kind of encompasses all the subprojects. What do you think?

**Comment by onbjerg** [comment]
If you're only using aragon/aragon for releases of the ÐApp, then why not move the entirety of the codebase into that repository as well? It seems a bit overhead-y to have "empty" repositories.

**Comment by izqui** [comment]
Yep, that was what I was talking to Luis. Migrate aragon-dapp to aragon, as
this repo is going to have less and less code and be an umbrella project
composed by multiple other submodules.

**Comment by Smokyish** [comment]
Yeah, i like the idea to migrate aragon-dapp to aragon and having issues in the repos, it should make it easier to track the issues.

If we move technical issues to the repos, I'm not sure if aragon/issues is needed, because Github shouldn't be a support platform, but used for the technicals, so you can concentrate on these issues and not have distracting things that would be better fitted elsewhere.

**Comment by izqui** [comment]
Progress:

- [x] Push all [aragon-dapp](https://github.com/aragon/aragon-dapp) code to [aragon/aragon](https://github.com/aragon/aragon).
- [x] Opened issues on aragon/aragon.
- [x] Added redirect.
- [x] Change all repo descriptions and purpose.
- [x] Updated READMEs.
